airflow/example_dags/example_xcom.py

```diff
@@ -79,8 +79,8 @@ def pull_value_from_bash_push(ti=None):
     bash_pull = BashOperator(
         task_id='bash_pull',
         bash_command='echo "bash pull demo" && '
-        f'echo "The xcom pushed manually is {bash_push.output["manually_pushed_value"]}" && '
-        f'echo "The returned_value xcom is {bash_push.output}" && '
+        f'echo "The xcom pushed manually is {XComArg(bash_push, key="manually_pushed_value")}" && '
+        f'echo "The returned_value xcom is {XComArg(bash_push)}" && '
```

Comment:
The latter one can still use `output` although using `XComArg` for both is probably more consistent…?

Reply:
Yeah, it might be better to be consistent. There are some other places I need to clean that up to use `XComArg()` throughout rather than `.output` for now. Is that cool?

Reply:
Although maybe as long as each file is consistent that's fine. `.output` is still a valid and useful API. I don't think we want to rip it out of all of the examples.